internal/gencli/method.go

```diff
@@ -255,6 +260,32 @@ func (c newMethodCommandCodeGenerator) generateFlag(
 	g.P(strconv.Quote(getFlagName(field, parents)), ",")
 	g.P(strconv.Quote(getFlagDescription(field)), ",")
 	g.P(")")
+	if resourceReference := proto.GetExtension(
+		field.Desc.Options(), annotations.E_ResourceReference,
```

Comment:
Since this now is open sourced, do we need to explain what `E_ResourceReference` is? Maybe a comment with a link?

Reply:
You know what. I thought this was something created by us, but then I looked in to where `annotations` come from and I realise it's not.

Reply:
Indeed! It's all standard issue :smile: